packages/tuono/src/dynamic/route_lazy_loading.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import type { ReactElement } from 'react'
+
+import type { RouteComponent } from 'tuono-router'
+
+type ImportFn = () => Promise<{ default: RouteComponent }>
+
+export const __tuono__internal__lazyLoadRoute = (
```

Comment:
How about rename the file to `RouteLazyLoading`  and give the internal name via export: 

```ts
export { RouteLazyLoading as __tuono__internal__lazyLoadRoute } from './dynamic/route_lazy_loading'
```